src/cli.ts

```diff
@@ -13,7 +13,7 @@ program
     .usage('[options] [yarn.lock path (default: yarn.lock)]')
     .option(
         '-s, --strategy <strategy>',
-        'deduplication strategy. Valid values: fewer, highest. Default is "highest"',
+        'deduplication strategy. Valid values: fewer, highest, direct.',
```

Comment:
commander already shows the default value so I removed `Default is "highest"` as it was redundant.